libs/components/pages/src/lib/modules/pipes/use-href.pipe.ts

```diff
@@ -0,0 +1,15 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'useHref',
+})
+export class UseHrefPipe implements PipeTransform {
```

Comment:
Can these pipes all be combined into one? it could either take a second parameter as the type you want to check (href, skyAppLink, etc.) and return a boolean, or you could keep the one parameter and return a string indicating which one to use.

Reply:
Consolidated to one pipe. Good call.